api/lib/application/error-manager.js

```diff
@@ -223,6 +223,9 @@ function _mapToHttpError(error) {
   if (error instanceof DomainErrors.UserCouldNotBeReconciledError) {
     return new HttpErrors.UnprocessableEntityError(error.message);
   }
+  if (error instanceof DomainErrors.CsvStudentsImportError) {
```

Comment:
Renommer cette erreur en
```suggestion
  if (error instanceof DomainErrors.CsvHigherSchoolingRegistrationImportError) {
```
?